deployment/helm/node-feature-discovery/values.yaml

```
@@ -429,3 +429,13 @@ rbac:
   ## Annotations for the Service Account
   ##
   serviceAccountAnnotations: {}
+
+# Optionally use encryption for worker <--> master comms
+# TODO: verify hostname is not yet supported
+# 
```

nit: trailing whitespace

```suggestion
#
```